### PR TITLE
Refactor:Move RssReader Fetch Article task to Sidekiq

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Refactor
- [x] Bug Fix

## Description
**Refactor:** Move the RssReader fetch job from a rake task to Sidekiq. 
**Bug Fix:** I took this chance to fix a quick bug we have been seeing in this task by skipping attempting to create articles if our feed object is missing a url. This error occurs thousands of times a day.
Fixes: https://app.honeybadger.io/projects/66984/faults/65532991
```
NoMethodError: undefined method `include?' for nil:NilClass
assembler.rb  63 thorough_parsing(...)
[PROJECT_ROOT]/app/services/rss_reader/assembler.rb:63:in `thorough_parsing'
61       html_doc = Nokogiri::HTML(content)
62       find_and_replace_possible_links!(html_doc) if @user.feed_referential_link
63       if feed_url.include?("medium.com")
64         parse_and_translate_gist_iframe!(html_doc)
65         parse_and_translate_youtube_iframe!(html_doc)
```
Screen Shot 2020-08-24 at 12 29 01 PM

## Related Tickets & Documents
https://github.com/orgs/forem/projects/6#card-43687049

## Added tests?
- [x] yes

alt_text

### CODE REVIEW

File: config/schedule.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+fetch_all_rss:
+  cron: "20 * * * *" # every hour, 20 min after the hour
```
Comment (maestromac) [inline config/schedule.yml:2]: Hopefully, this is enough time and won't bring back the SIGTERM trouble we had in the past.

Comment (mstruve) [reply config/schedule.yml:2]: I matched what was in the scheduler so I would think so. 
Screen Shot 2020-08-24 at 3 04 15 PM

Comment (maestromac) [reply config/schedule.yml:2]: Oh sorry, I misread that, I thought it says every 20 min.

**Comment by mstruve** [comment]
The additional files are linter fixes when I merged with master.

**Review by rhymes — APPROVED** [review]
it appears there's some unrelated code but it's formatting so LGTM :D

**Comment by mstruve** [comment]
@rhymes yeah some linter fixes from merging with master and having to edit a file in the merge commit. I guess I should use `--no-verify` when I do that so the rubocop fixes dont run.

**Pull request merged**